Gems/Atom/Feature/Common/Code/Source/PostProcessing/EyeAdaptationPass.cpp

```diff
@@ -73,8 +73,6 @@ namespace AZ
                 return false;
             }
 
-            AZ_Assert(m_pipeline->GetScene(), "EyeAdaptationPass's Pipeline does not have a valid scene pointer");
```

Comment:
Why remove this?  rare edge cases like that can happen so it'd be a good indication before crashing.

Reply:
I think we don't need 150+ passes each checking that the pipeline has a scene. The pipeline should enforce this itself.